src/hooks/Tokens.ts

```diff
@@ -28,8 +29,36 @@ function useTokensFromMap(tokenMap: TokenAddressMap, chainId: Maybe<SupportedCha
   }, [chainId, tokenMap])
 }
 
-export function useAllTokensMultichain(): TokenAddressMap {
-  return useCombinedTokenMapFromUrls(DEFAULT_LIST_OF_LISTS)
+export type ChainTokenMap = { [chainId in number]?: { [address in string]?: Token } }
```

Comment:
How is this different from useDefaultActiveTokens? Could you just reuse that instead to i improve code reuse?

Reply:
`DefaultActiveTokens` are tokens on the default list + user added tokens. 

`AllTokensMultichain` are tokens from any list + user added tokens